src/include/migraphx/value.hpp

```diff
@@ -122,6 +122,7 @@ struct value
 // clang-format off
 #define MIGRAPHX_VISIT_VALUE_TYPES(m) \
     m(int64, std::int64_t) \
+    m(uint8, std::uint8_t) \
```

Comment:
Why is this added?

Reply:
I would prefer to have only one signed and one unsigned integer type in the value class.

Reply:
It was required to run the python test with type = "uint8".

Reply:
without having that, it is not possible to convert value object of "uint8" type to string object.  string is then passed to 'shape::parse_type()`.

Reply:
> convert value object of "uint8" type to string object

I am not sure I understand. How are you creating a value object of "uint8" when it doesn't exist?

Reply:
For example : 
s = migraphx.shape(type='uint8_t')

in `migraphx_py.cpp` this is passed in as `py::kwargs` and those kwargs are converted into value object where type is one of the memeber. {{"type", "uint8"}}.  uint8 is converted into string from there and passed to parse_type(). In order to convert to string, it requires `uint8` to be added to one of the `MIGRAPHX_VISIT_VALUE_TYPES`.

Reply:
>  uint8 is converted into string from there and passed to parse_type(). In order to convert to string, it requires uint8 to be added to one of the MIGRAPHX_VISIT_VALUE_TYPES.

It shouldn't need `uint8` to convert to string. It should already be converted to a string in `visit_py`.

Reply:
> > uint8 is converted into string from there and passed to parse_type(). In order to convert to string, it requires uint8 to be added to one of the MIGRAPHX_VISIT_VALUE_TYPES.
> 
> It shouldn't need `uint8` to convert to string. It should already be converted to a string in `visit_py`.

"uint8" was converted into string in `visit_py`.  But for some reason during `visit` in `value.hpp`, `this->get_type()` resulted in `uint8_type` instead of `string_type`.  I am not sure why it was that way.  Adding `uint8` into `MIGRAPHX_VISIT_VALUE_TYPES` then added case for `uint8_type` and everything worked for some reason.

Reply:
Removed that change from `value.hpp`.  Please disregard comments i made about the visit etc in value.hpp.